src/plugins/intel_gpu/tests/unit/test_cases/reorder_gpu_test.cpp

```diff
@@ -2142,6 +2143,103 @@ TEST(reorder_weights_gpu_i32, reorder_weights)
+
+    auto constant_1 = engine.allocate_memory(layout(ov::PartialShape{ 1, 192, 1 }, data_types::f32, format::bfyx));
+    auto constant_1_values = rg.generate_random_1d<float>(constant_1->count(), -1, 1);
+    set_values(constant_1, constant_1_values);
+
+    auto input = engine.allocate_memory(layout(ov::PartialShape{ 1, 192, 91 }, data_types::f32, format::bfyx));
+    auto input_values = rg.generate_random_1d<float>(input->count(), -1, 1);
+    set_values(input, input_values);
+
+    auto constant_0 = engine.allocate_memory(layout(ov::PartialShape{ 1, 1, 91 }, data_types::f32, format::bfyx));
+    auto constant_0_values = rg.generate_random_1d<float>(constant_0->count(), -1, 1);
+    set_values(constant_0, constant_0_values);
+
+    topology topology;
+    topology.add(data("shape", shape));
+    topology.add(data("min_val", min_val));
+    topology.add(data("max_val", max_val));
+    topology.add(random_uniform(
+                 "random_uniform",
+                 {
+                     input_info("shape"),
+                     input_info("min_val"),
+                     input_info("max_val")
+                 },
+                 data_types::f32,
+                 150,
+                 10,
+                 ov::Shape{ 1, 1, 91 }));
+    topology.add(data("weights", weights));
+    topology.add(convolution(
+                 "convolution",
+                 input_info("random_uniform"),
+                 "weights",
+                 "",
+                 1,
+                 { 1 },
+                 { 1 },
+                 { 0 },
+                 { 0 },
+                 false));
+    topology.add(data("constant_1", constant_1));
+    topology.add(eltwise("add_1",
+                 {
+                    input_info("convolution"),
+                    input_info("constant_1")
+                 },
+                 eltwise_mode::sum));
+    topology.add(input_layout("parameter", input->get_layout()));
+    topology.add(data("constant_0", constant_0));
+    topology.add(eltwise("multiply",
+                 {
+                    input_info("parameter"),
+                    input_info("constant_0")
+                 },
+                 eltwise_mode::prod));
+    topology.add(eltwise("add_0",
+                 {
+                    input_info("add_1"),
+                    input_info("multiply")
+                 },
+                 eltwise_mode::sum));
+    topology.add(reorder("result",
+                 input_info("add_0"),
+                 format::bfyx,
+                 data_types::f32));
+
+    ExecutionConfig config = get_test_default_config(engine);
+    config.set_property(ov::intel_gpu::allow_new_shape_infer(true));
+    config.set_property(ov::intel_gpu::optimize_data(true));
+
+    network network(engine, topology, config);
+    network.set_input_data("parameter", input);
+
+    auto outputs = network.execute();
+    ASSERT_EQ(outputs.size(), size_t(1));
+    ASSERT_EQ(outputs.begin()->first, "result");
```

Comment:
What this test intends to test for? IT is just checking # of output and result node name....